packages/react-components/lib/tasks/task-table.tsx

```diff
@@ -15,26 +20,34 @@ const useStyles = makeStyles((theme) => ({
     background: theme.palette.action.hover,
   },
   infoRow: {
+    boxShadow: `${theme.shadows[1]}`,
     '& > *': {
       borderBottom: 'unset',
     },
   },
   phasesCell: {
-    padding: `0 ${theme.spacing(1)}px`,
+    padding: `0 ${theme.spacing(1)}px 0 ${theme.spacing(1)}px`,
+    boxShadow: `${theme.shadows[1]}`,
+    '&:last-child': {
+      paddingRight: `${theme.spacing(1)}px`,
+    },
   },
   phasesRow: {
     marginBottom: theme.spacing(1),
+    marginTop: theme.spacing(1),
   },
 }));
```

Comment:
yea! but I was thinking we could have a separate story / component using the phases and keep it as an option within react-components

Reply:
The phases is in its own component, I don't think it is worth it to create different variants of the table, not unless we actually have a use case for different variants, better to keep things simple to avoid bloating it.